src/main/java/robot/subsystems/drivetrain/pure_pursuit/Constants.java

```diff
@@ -24,9 +24,17 @@
     public final static double kA = 0.002;
     public final static double kP = 0.01;
 
+    // fall control
+    public final static double Kj = 0;
```

Comment:
What does `Kj` mean? give it a better name.
Also, If it is a constant, it should be `ALL_CAPS_WITH_UNDERSCORES`. Else, it should be `camelCase`.

Reply:
the jerk constant and  i fixed this